registry/tests/appRoutes.spec.ts

```diff
@@ -244,6 +280,35 @@ describe(`Tests ${example.url}`, () => {
             }
         });
 
+        it('should create record with existed domainId', async () => {
+            const domainId = await createRouterDomain(example.routerDomain);
+
+            const exampleWithExistedDomainId = {
+                ...example.correct,
+                domainId,
+            };
+
+            let response = await request.post(example.url)
+                .send(exampleWithExistedDomainId)
+            .expect(200);
+
+            expect(response.body).to.have.property('id');
+            const id = response.body.id;
+
+            try {
+                const expectedRoute = { id, ..._.omitBy(exampleWithExistedDomainId, _.isNil) };
+                expect(response.body).deep.equal(expectedRoute);
+
+                response = await request.get(example.url + id)
+                    .expect(200);
+
+                expect(response.body).deep.equal(expectedRoute);
+            } finally {
+                await request.delete(example.url + id);
+                await request.delete(example.routerDomain.url + domainId).expect(204);
```

Comment:
rm expect here, it's not necessary

Reply:
but what if `delete` is broken? in this case we will face the problem with the next `it`, instead of current

Reply:
We should have an explicit test case for delete operation instead of this implicit check, IMO

Reply:
done